src/notation/internal/notationinteraction.cpp

```diff
@@ -72,9 +73,8 @@ NotationInteraction::NotationInteraction(Notation* notation, INotationUndoStackP
         }
     });
 
-    m_dragData.ed.view = new ScoreCallbacks();
-    m_dropData.ed.view = new ScoreCallbacks();
-    m_gripEditData.view = new ScoreCallbacks();
+    m_dragData.ed = Ms::EditData(&m_scoreCallbacks);
+    m_dropData.ed = Ms::EditData(&m_scoreCallbacks);
```

Comment:
Tiny bit inefficient but this only gets created rarely

Reply:
A bit unreliable, potential crash is possible if we pass a reference to a member of one class to another class. (for example, if in the future `m_dragData` and `m_scoreCallbacks` are swapped in the .h file and there will be some call to `m_scoreCallbacks` in the `EditData` destructor)
Maybe it's better to do a `shared_ptr`?
It will also be more correct semantically - the semantics of the code will "say" that `m_scoreCallbacks` is a shared object.